desktop_version/src/Script.cpp

```diff
@@ -519,9 +520,13 @@ void scriptclass::run(void)
                 textx = ss_toi(words[2]);
                 texty = ss_toi(words[3]);
 
+                // If words[4] ends with L, set large to true.
+                textlarge = (words[4].size() > 0 && words[4][words[4].size() - 1] == 'l');
+                int lines = ss_toi(words[4].substr(0, words[4].size() - (textlarge ? 1 : 0)));
```

Comment:
You don't need to do any complicated `substr` stuff because `ss_toi` will take in every numeric character until it encounters one that isn't a numeral. Though I agree with keeping it out of the for-loop conditional so it doesn't get evaluated on every iteration.
```suggestion
                int lines = ss_toi(words[4]);
```